app/handlers/ErrorHandler.scala

```diff
@@ -43,7 +43,7 @@ class ErrorHandler @Inject()(appConfig: AppConfig, val messagesApi: MessagesApi,
 
   override def resolveError(rh: RequestHeader, ex: Throwable): Result =
     ex match {
-      case _: NoActiveSession => toGGLogin(rh.uri)
```

Comment:
`toGGLogin` only works for production as far as I'm aware. You can override the host but that isn't sufficient as we need to redirect to the auth-stub